modules/go-add-infra-nodes.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="add-infra-nodes_{context}"]
+= Adding infrastructure nodes
+
+.Procedure
+
+. Label existing nodes as infrastructure by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra
+----
+. Edit the `GitOpsService` Custom Resource (CR) to add the infrastructure node selector:
++
+[source,terminal]
+----
+$ oc edit gitopsservice -n openshift-gitops
+----
+. In the `GitOpsService` CR file, add `runOnInfra` toggle to the `spec` section and set it to `true`:
++
+[source,yaml]
+----
+apiVersion: pipelines.openshift.io/v1alpha1
+kind: GitopsService
+metadata:
+  name: cluster
+spec:
+  runOnInfra: true
+----
+. Optional: Apply taints and isolate the workloads on infrastructure nodes and prevent other workloads from scheduling on these nodes. 
++
+[source,terminal]
+----
+$ oc adm taint nodes -l node-role.kubernetes.io/infra 
+infra=reserved:NoSchedule infra=reserved:NoExecute
+----
++
+. Optional: If you apply taints to the nodes, you can add tolerations in the `GitOpsService` CR:app-name:
++
+[source,yaml]
+----
+spec:
+  runOnInfra: true
+  tolerations:
+  - effect: NoSchedule
+    key: infra
+    value: reserved
+  - effect: NoExecute
+    key: infra
+    value: reserved 
+----
+
+To verify that the workloads are scheduled on infrastructure nodes in the {gitops-title} namespace, click any of the pod names and ensure that the *Node selector* and *Tolerations* have been added.
```

Comment:
The {gitops-title} attribute does not render in the preview, pls verify that the include for the gitops attribute file has been added to the assembly, looks like it is missing.
![Screenshot from 2022-01-07 19-08-18](https://user-images.githubusercontent.com/16647625/148551951-188831e6-f4d4-4f9e-b86c-5ea888f3d452.png)